runelite-client/src/main/java/net/runelite/client/plugins/runecraft/RunecraftOverlay.java

```diff
@@ -0,0 +1,41 @@
+package net.runelite.client.plugins.runecraft;
```

Comment:
I feel like this would be better as a widgetitemoverlay, which only gets added when a pouch is degraded